petclinic-frontend/src/features/inventories/api/global.d.ts

```diff
@@ -0,0 +1,35 @@
+declare global {
+  interface GoogleTranslateOptions {
+    pageLanguage: string;
+    autoDisplay: boolean;
+    includedLanguages: string;
+    layout?: number;
+  }
+
+  interface TranslateElement {
+    new (
+      options: GoogleTranslateOptions,
+      elementId: string
+    ): TranslateElementInstance;
+  }
+
+  interface TranslateElementInstance {}
+
+  interface GoogleTranslate {
+    TranslateElement: TranslateElement;
+    InlineLayout: {
+      SIMPLE: number;
+      HORIZONTAL: number;
+      VERTICAL: number;
+    };
+  }
+
+  interface Window {
+    google: {
+      translate: GoogleTranslate; // Google Translate API reference
+    };
+    googleTranslateElementInit: () => void; // Initialization function for Google Translate
+  }
+}
+
+export {};
```

Comment:
Impressive use of google translate for every element of the UI of inventories. You seem to have covered every functionality linked to the Google Translate API.